cmd/gen-go-sample/main.go

```diff
@@ -271,6 +272,40 @@ func (g *generator) genSample(ifaceName string, methConf GAPICMethod, regTag str
 
 	// Some parts of request object are from arguments.
 	for _, attr := range valSet.Parameters.Attributes {
+		if attr.ReadFile {
+			varName, err := fileVarName(attr.Parameter)
```

Comment:
You're performing this non-trivial calculation which may just get thrown away.

For clarity, I would 
```
var varName
if attr.SampleArgumentName != "" {
  varName = snakeToCamel(...)...
} else {
  varName = fileVarName(...)
}
```